packages/snap/src/utils/types.ts

```diff
@@ -0,0 +1,20 @@
+import * as t from 'io-ts'
+
+export const SimpleTransaction = t.intersection([
```

Comment:
I would name these `SimpleTransactionSchema` and `RequestParamsSchema` to distinguish from the actual types, which should also be defined here

Reply:
Also, did you make any progress on determining what type we should expect from the Dapp representing the transaction?

Reply:
SimpleTransaction is just meant to be a placeholder. @aaronmgdr maybe you have a suggestion?

Reply:
Yes, you should first determine if CeloTx is the right type to expect from the dapp and then write an io-ts schema for it